content/pages/index/content.json

```diff
@@ -2,19 +2,19 @@
   "content": {
     "teaser": {
       "title": "Gaia-X: Data Sovereignty for Europe",
-      "text": "Do you want to learn how a Gaia-X build on Distributed Ledger Technology can be utilized by your company? Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+      "text": "Gaia-X represents the next generation of data infrastructure: an **open, transparent and secure digital ecosystem**, where data and services can be made available, collated and shared in an environment of trust. The architecture of Gaia-X is based on the principle of decentralisation."
     },
     "paragraphs": [
       {
         "title": "A Minimal Viable Gaia-X on DLT",
-        "body": "Do you want to learn how a Gaia-X build on Distributed Ledger Technology can be utilized by your company? Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+        "body": "The principle of decentralisation is a perfect fit for **Distributed Ledger Technology**. [Ocean Protocol](https://oceanprotocol.com) is a fully open-source technology stack, leveraging DLT and decentralization, aiming to give control back to the owners of data services. This MVG demonstrator is built on top of Ocean Protocol showcasing how a Minimal Viable Gaia-X on DLT and Ocean Protocol can look like, already today.",
```

Comment:
issue, question(blocking): I am not sure but I think we are using American English spelling (decentralization).